Code/Framework/AzCore/AzCore/Component/EntitySerializer.cpp

```diff
@@ -81,9 +81,14 @@ namespace AZ
                     azrtti_typeid<decltype(componentMap)>(),
                     inputValue, "Components", context);
 
+            static TypeId genericComponentWrapperTypeId("{68D358CA-89B9-4730-8BA6-E181DEA28FDE}");
```

Comment:
It would be good to not have this hardcoded here. Is there a header that declares this we could use? We should be able to do azrtti_typeid<GenericComponentWrapper>() correct?

Reply:
Hmm, this could be solved by creating a static typeId var in AZ::Component or nearby and using that value in the AzFramework defination of GenericComponentWrapper. Allowing both to have access. We've seen similar patterns for Transform.

However that's likely out of scope for this change. So we can keep as is for now.